src/python3/cpp_simulator.cpp

```diff
@@ -512,6 +512,30 @@ class DataWrangler
 };
 
 
+template<size_t dim>
+struct PyParticle
```

Comment:
we already have a `Particle` object in pharesee, somehow this creates confusion.
can we reconcile them?

Reply:
to be considered
however:
Pharesee doesn't know about pybind C++ stuff.
Getting Python Objects into C++ is possible, but not so fun.
Simplest might be to take Pharesee internals as function parameters.

Reply:
removed